liquidator/liquidator.js

```diff
@@ -58,7 +51,8 @@ class Liquidator {
         at: "liquidator",
         message: "liquidating sponsor 🔥",
```

Comment:
unrelated to your PR but
```suggestion
        message: "liquidating sponsor🔥",
```